app/src/main/res/layout/options_without_drawer_activity.xml

```diff
@@ -8,9 +9,15 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <include
-      android:id="@+id/options_activity_toolbar"
-      layout="@layout/toolbar" />
+      <androidx.appcompat.widget.Toolbar
+        android:id="@+id/options_activity_toolbar"
```

Comment:
And ditto here.

Reply:
Renaming the id triggers an avalanche of test fails as show above PTAL